<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommandsOneWireOldWindow</class>
 <widget class="QMainWindow" name="CommandsOneWireOldWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>461</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>689</width>
    <height>461</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>689</width>
    <height>461</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 200, 200);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="vddButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Питание</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newPartButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Новая партия</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addArchiveButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить в архив</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startWorkButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>70</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 41, 209);</string>
    </property>
    <property name="text">
     <string>предваритальная
 настройка</string>
    </property>
   </widget>
   <widget class="QPushButton" name="readTempButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(152, 255, 176);</string>
    </property>
    <property name="text">
     <string>температура</string>
    </property>
   </widget>
   <widget class="QPushButton" name="readAddressButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(152, 255, 176);</string>
    </property>
    <property name="text">
     <string>address</string>
    </property>
   </widget>
   <widget class="QPushButton" name="readOTPButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>210</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(152, 255, 176);</string>
    </property>
    <property name="text">
     <string>OTP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="writeKAndBButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>140</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 183, 180);</string>
    </property>
    <property name="text">
     <string>K и B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="write3VButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>280</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 183, 180);</string>
    </property>
    <property name="text">
     <string>3V</string>
    </property>
   </widget>
   <widget class="QPushButton" name="writeENButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>210</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 183, 180);</string>
    </property>
    <property name="text">
     <string>EN и Отключение DQ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>360</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(156, 179, 255);</string>
    </property>
    <property name="text">
     <string>измерение</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
